projects/aca-folder-rules/src/lib/rule-details/actions/rule-action.ui-component.ts

```diff
@@ -200,6 +224,45 @@ export class RuleActionUiComponent implements ControlValueAccessor, OnInit, OnDe
     });
   }
 
+  openSelectorDialog() {
+    const data: ContentNodeSelectorComponentData = {
+      title: this.translate.instant('ACA_FOLDER_RULES.RULE_DETAILS.PLACEHOLDER.CHOOSE_FOLDER'),
+      actionName: NodeAction.CHOOSE,
+      currentFolderId: this._nodeId,
+      select: new Subject<any>()
```

Comment:
is that possible to use different type than any?

Reply:
Changed, thanks